app/components/elements/SlideShow/SlideShow.js

```diff
@@ -0,0 +1,262 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Box, BoxProps, Flex, Image } from 'theme-ui';
+import map from 'lodash/map';
+import { useTranslation } from 'react-i18next';
+import { useSnapCarousel } from 'react-snap-carousel';
```

Comment:
There were a couple other carousel libraries that met our needs of being lightweight, responsive, accessible, and supporting css-based snappoint dragging, while being a bit higher level with predefined components, but this was the only one that supported React 16.

This one works well though.  Just a bit more verbose/low-level than I'd wanted.